scalableminds/webknossos
# render docs via wk-libs using mkdocs

### Steps to test:
- Check the new docs as rendered with https://github.com/scalableminds/webknossos-libs/pull/422:
  https://static.webknossos.org/lib-docs/integrate_wk_docs/webknossos/index.html

This should only be merged after https://docs.webknossos.org/ is deployed via webknossos-libs
